### PR TITLE
docs(deployment): Suggested wording changes for schema.yaml.

### PR DESCRIPTION
**Pull request by joeliedtke**
**Description of your changes:**
Updated schema.yaml with wording and branding changes.

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention). 

   PR titles examples:
    * `fix(frontend): fixes empty page. Fixes #1234`
       Use `fix` to indicate that this PR fixes a bug.
    * `feat(backend): configurable service account. Fixes #1234, fixes #1235`
       Use `feat` to indicate that this PR adds a new feature. 
    * `chore: set up changelog generation tools`
       Use `chore` to indicate that this PR makes some changes that users don't need to know.
    * `test: fix CI failure. Part of #1234`
        Use `part of` to indicate that a PR is working on an issue, but shouldn't close the issue when merged.

- [x] Do you want this pull request (PR) cherry-picked into the current release branch?
    
    If yes, use one of the following options:
  
    * **(Recommended.)** Ask the PR approver to add the `cherrypick-approved` label to this PR. The release manager adds this PR to the release branch in a batch update.
    *  After this PR is merged, create a cherry-pick PR to add these changes to the release branch. (For more information about creating a cherry-pick PR, see the [Kubeflow Pipelines release guide](https://github.com/kubeflow/pipelines/blob/master/RELEASE.md#option--git-cherry-pick).)

### CODE REVIEW

File: manifests/gcp_marketplace/schema.yaml [hunk 1]
```diff
@@ -118,69 +118,79 @@ properties:
...
-      This field must be specified if choose to use managed storage.
-      Provide the instance connection name for an existing Cloud SQL for MySQL instance.
-      The instance connection name can be found on the instance detail page in the Cloud SQL console.
-      The instance connection name uses the format project:zone:instance-name, for example,myproject:us-central1:myinstance.
-      For more details on how to create a new instance, see https://cloud.google.com/sql/docs/mysql/quickstart.
+      If you are deploying Kubeflow Pipelines with managed storage, specify the
+      instance connection name for your MySQL instance on Cloud SQL.
+      The instance connection name can be found on the instance detail page in
+      the Cloud SQL console. The instance connection name uses the format
+      "<project>:<zone>:<instance-name>", for example: myproject:us-central1:myinstance.
```
Comment (Bobgy) [inline manifests/gcp_marketplace/schema.yaml:159]: We have a mistake here, it should be "\<project>:\<region>:\<instance-name>", us-central1 is a region. (verified in real data, this is always a region)

Comment (Bobgy) [reply manifests/gcp_marketplace/schema.yaml]: UPDATE: I edited the PR in-place to change this to region

Comment (rmgogogo) [reply manifests/gcp_marketplace/schema.yaml]: good point, thanks Yuan.

**Comment by rmgogogo** [comment]
/lgtm

Thanks Joe. May fix the comment of region/zone and then we can submit

**Comment by Bobgy** [comment]
I've fixed the region part.
let's submit
/lgtm
/approve

**Pull request merged**